### ISSUE TITLE
"before all" hook Uncaught SecrityError: localStorage is not available for opaque origins

### ISSUE THREAD

**Issue by guysbryant** [learn-co-curriculum/javascript-hide-and-seek#18]
Here is what is returned in my online IDE console:

```[10:08:06] ~
// ♥ learn open javascript-hide-and-seek-bootcamp-prep-000
Looking for lesson...
Forking lesson...
Cloning lesson...
Opening lesson...
Installing npm dependencies...
yarn install v1.7.0
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
warning Your current version of Yarn is out of date. The latest version is "1.9.4", while you're on "1.7.0".
info To upgrade, run the following command:
$ sudo apt-get update && sudo apt-get install yarn
Done in 7.60s.
Done.
[10:08:17] (master) javascript-hide-and-seek-bootcamp-prep-000
// ♥ learn

> java-script-hide-and-seek@0.1.0 test /home/guysbryant/javascript-hide-and-seek-bootcamp-prep-000
> mocha -R mocha-multi --reporter-options nyan=-,json=.results.json

 0   __,------,
 1   __|  /\_/\
 0   _~|_( x .x)
     _ ""  ""

  0 passing (761ms)
  1 failing

  1) "before all" hook:
     Uncaught SecurityError: localStorage is not available for opaque origins
      at Window.get localStorage [as localStorage] (node_modules/mocha-jsdom/node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
      at propagateToGlobal (node_modules/mocha-jsdom/index.js:108:27)
      at Object.done (node_modules/mocha-jsdom/index.js:59:9)
      at process.nextTick (node_modules/mocha-jsdom/node_modules/jsdom/lib/old-api.js:366:18)
      at _combinedTickCallback (internal/process/next_tick.js:131:7)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! Test failed.  See above for more details.
[10:08:23] (master) javascript-hide-and-seek-bootcamp-prep-000
// ♥

**Comment by cnezames** [comment]
In package.json change this: "mocha-jsdom": "^1.1.0", to "mocha-jsdom": "~1.1.0", and run npm install

**Comment by dafuloth** [comment]
And I was about to create a new issue regarding this... thanks for providing the fix here, @cnezames 

IMO, issue should remain open until either the Lab files are updated with the fix or a note is added to the Lab page describing this fix so that people know to apply the fix themselves.

**Comment by victoria-huang** [comment]
After changing to "mocha-jsdom":  "~1.1.0", we received this error: TypeError: require(...).env is not a function (in the mocha-jsdom index.js)

**Comment by victoria-huang** [comment]
Changing this in package.json fixed the issue for us: "jsdom": "^11.11.0" to "jsdom": "11.11.0"

**Comment by maxwellbenton** [comment]
Thank you for spotting this issue and providing feedback, @guysbryant.

We have updated the materials and believe your issue to have been resolved.

We apologize for any frustration you might have encountered during this process but thank you for helping us ensure that those who follow in your path will not encounter the same problems that you faced.

If you do not believe that this issue has been addressed, please re-open this issue. 💙
